### DISCUSSION TITLE
How to include tabbed pages inside a FlyOut Page

### DISCUSSION THREAD

**Discussion by IXChinmayD** [dotnet/maui#23842] [Q&A]
I wanted to show tabbed pages inside a FlyOut Page. Is there any documentation?

**Comment by axa88** [comment]
> Is there any documentation?

Yes.
https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/flyoutpage
https://learn.microsoft.com/en-us/dotnet/maui/user-interface/pages/tabbedpage

otherwise, hows your Portuguese?
https://www.youtube.com/watch?v=T3HPTy86rU4

Or if you despise xml like me, and dont want to google:

I assume you want the `Tabbed `pages on the `Detail` side of the `FlyoutPage` and not the `Flyout` itself...
```
public class MyFlyoutPage : FlyoutPage
{
	public MyFlyoutPage()
	{
		// without setting Titles it will throw
		Flyout = new ContentPage() { Title = "flyout page"};
		Detail = new TabbedPage()  { Title = "detail page"};
}

public class MyTabbedPage
{
	public MyTabbedPage()
	{
		Children.Add(new ContentPage());
		Children.Add(new ContentPage());
	}
}
```
